bin/ember

```diff
@@ -4,6 +4,7 @@
 // Provide a title to the process in `ps`
 process.title = 'ember';
 
+// require('time-require');
```

Comment:
that would yet again increase the `npm install` time just for debugging purposes. I'm not really a huge of that.